ui/app/templates/components/console/log-help.hbs

```diff
@@ -6,6 +6,7 @@
 
 Commands:
   read        Read data and retrieves secrets
+  kv-read     Read data for kv v2 secret engines. Use -metadata flag to read metadata
```

Comment:
I'm curious of the reason behind deviating from the vault CLI commands and using `read` here instead of `get`?